assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/ticketMainMenuItem/CopyTicket.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react';
+import { pick } from 'ramda';
+import { __ } from '@wordpress/i18n';
+
+import { Copy } from '@application/ui/layout/entityActionsMenu/entityMenuItems';
+import { useTicketContext } from '@edtrHooks/index';
+import { useTicketItem } from '@edtrServices/apollo/queries';
+import { useTicketMutator } from '@edtrServices/apollo/mutations';
+import { useRelations } from '@appServices/apollo/relations';
+
+const CopyTicket: React.FC = () => {
+	const { id } = useTicketContext();
+	const ticket = useTicketItem({ id });
+	const { createEntity } = useTicketMutator();
+	const { getRelations } = useRelations();
+	const newTicket = pick(
+		['capacity', 'description', 'endDate', 'isPrimary', 'name', 'order', 'reserved', 'sold', 'startDate'],
+		ticket
+	);
+	const datetimes = getRelations({
+		entity: 'tickets',
+		entityId: id,
+		relation: 'datetimes',
+	});
+	const title = __('copy ticket');
```

Comment:
imho simple one time use things like this can just be used directly in the component declaration (ie: ` title={ __('copy ticket') } `)
just like you have done in `TicketMainMenuItem` below

Reply:
done